src/components/frame/Extensions/layouts/PageLayout.tsx

```diff
@@ -58,8 +64,11 @@ const ExtensionsLayout: FC<PropsWithChildren> = ({ children }) => {
     applyTheme(isDarkMode ? Theme.dark : Theme.light);
   }, [isDarkMode]);
 
+  // @TODO: Handle other pages when needed or handle navigation in the page itself
+  const pageName = loadingText === 'extensionPage' ? 'extensions' : 'members';
```

Comment:
Shouldn't be a translation here?

Reply:
It's only used as id, so we don't need translation here